### PR TITLE
Rename marketing team to contributor comms in `sigs.yaml` and fix some typos

### PR DESCRIPTION
**Pull request by SD-13**
This PR updates the `contributor marketing` team with the new name `Contributor Comms` in sigs.yaml.
Also, it fixes some typos in the [storytelling blog guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/blog-guidelines.md) doc

### CODE REVIEW

**Comment by kaslin** [comment]
/ok-to-test
Thanks for submitting a PR for this!
/lgtm

**Comment by SD-13** [comment]
Hi @kaslin, sorry for the trouble. Now it's ready for review. please take a look when you are free. Thanks,

File: communication/contributor-comms/storytelling-resources/blog-guidelines.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ This process is initiated in [#sig-contribex-comms](https://kubernetes.slack.com
...
 3. Create your proposal draft in [Google Docs](https://docs.google.com/) or HackMD (https://hackmd.io), and ask for a review in [the channel](https://kubernetes.slack.com/archives/C03KT3SUJ20). This will facilitate easier editing, especially if major changes or restructuring is needed. Take into account the [documentation style guide](https://kubernetes.io/docs/contribute/style/style-guide/): these guidelines can help in improving the readability of your article, especially in terms of the use of Kubernetes terminology.
 4. Once you have reflected any feedback in the proposal draft,  announce that the article is ready for submission (again, in the channel or in one of the weekly meetings): the assigned editor will use the final text to open the PR, adding you as a Co-author
-5. (Optional) If the submission will be mirrored in the main Kubernetes site, a second PR will be oped by the editor, but on the main repository, and after being approved in the `contributor-site` one.
+5. (Optional) If the submission will be mirrored in the main Kubernetes site, a second PR will be opend by the editor, but on the main repository, and after being approved in the `contributor-site` one.
```
Comment (palnabarun) [inline communication/contributor-comms/storytelling-resources/blog-guidelines.md:33]: nit
```suggestion
5. (Optional) If the submission will be mirrored in the main Kubernetes site, a second PR will be opened by the editor, but on the main repository, and after being approved in the `contributor-site` one.
```

Comment (SD-13) [reply communication/contributor-comms/storytelling-resources/blog-guidelines.md]: done

**Comment by SD-13** [comment]
Thanks @palnabarun for reviewing the PR!

**Pull request merged**